envs/development/conda/fedbiomed-researcher.yaml

```diff
@@ -10,63 +10,59 @@ channels:
 
 dependencies:
   # minimal environment
-  - python >=3.9,<3.10
-  - pip
-  - jupyter
-  - ipython
+  - python >=3.10,<3.11
+  - pip >= 23.0
+  - jupyter ~=1.0.0
```

Comment:
Hi @mvesin,

It seems that Jupyter classic notebooks won't be maintained anymore. https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html they recommend to migrate Notebook 7  and JupyterLab instead of notebooks. We may create another issue for that but I am adding this comment since this pull request is about upgrading packages.

Reply:
Good point.
I noted this as a comment in the `yml` files, but will not treat it as a part of this PR (at least needs to redo testings).
My understanding is we also need to choose between classical notebooks v7 and migrating to jupyterlab.

Issue #809 is created for it.